0-email-signup-form/FCCvote-net-neutrality-thanks/below-form-content.html

```diff
@@ -24,7 +27,7 @@
   // ***********************************************
   $(window).on("load", function () {
     $bsdForm.remove();
-    var carriedEmail = $.urlParam("email");
+    var carriedEmail = queryParams["email"];
```

Comment:
You can directly tap into `queryParams.email` here, since it's an object. It alos means you can test for `if (!queryParams.email) { this is a problem }`